mlir/utils/jenkins/Jenkinsfile

```diff
@@ -35,10 +35,11 @@ void showEnv() {
     sh 'ulimit -a'
     sh '/opt/rocm/bin/rocm-smi'
     sh '/opt/rocm/bin/rocm_agent_enumerator'
+    sh 'id'
 }
 
 String dockerArgs() {
-    return '--user "$(id -u):$(id -g)" --device=/dev/kfd --device=/dev/dri --group-add video -u 0'
+    return "--device=/dev/kfd --device=/dev/dri --group-add video -v /etc/passwd:/etc/passwd:ro -v /etc/group:/etc/group:ro"
```

Comment:
I'm not sure we need to pass in /etc/passwd and /etc/group - what happens if we don't do that?

Reply:
I got something like this:
"groups: cannot find name for group ID"
"I have no name!"

Reply:
What command does this come from?

Reply:
From the docker command without -v

Reply:
Makes sense, I can see why we'd want to bind mount in this info